core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -437,7 +438,7 @@ object KafkaConfig {
   val LeaderImbalancePerBrokerPercentageDoc = "The ratio of leader imbalance allowed per broker. The controller would trigger a leader balance if it goes above this value per broker. The value is specified in percentage."
   val LeaderImbalanceCheckIntervalSecondsDoc = "The frequency with which the partition rebalance check is triggered by the controller"
   val UncleanLeaderElectionEnableDoc = "Indicates whether to enable replicas not in the ISR set to be elected as leader as a last resort, even though doing so may result in data loss"
-  val InterBrokerSecurityProtocolDoc = "Security protocol used to communicate between brokers. Defaults to plain text."
+  val InterBrokerSecurityProtocolDoc = s"Security protocol used to communicate between brokers. Valid values are: ${SecurityProtocol.getNames.asScala.mkString(", ")}."
```

Comment:
Since normal users shouldn't set TRACE, so it seems that we shouldn't include it in the docs, which is for the users.